### PR TITLE
[doc][download]Fix download link not found in document

### PR DESCRIPTION
**Pull request by tuteng**
Fixes https://github.com/apache/pulsar/issues/4712

Master Issue: https://github.com/apache/pulsar/issues/4712

### Motivation
The download link of the current document about debezium is lost, resulting in that the user cannot find the correct download address in the document.
image

### Modifications

Add download link of debezium MySQL and PostgreSQL.

### CODE REVIEW

File: site2/website/data/connectors.js [hunk 1]
```diff
@@ -5,6 +5,18 @@ module.exports = [
         type: 'Sink',
         link: 'https://www.aerospike.com/'
     },
+    {
```
Comment (sijie) [inline site2/website/data/connectors.js]: I think the list is sorted in alphabet order. so you have to add them to be after `data-generate` and before `elastic-search`.

**Comment by tuteng** [comment]
run java8 tests
run Cpp tests

**Comment by sijie** [comment]
run cpp tests

**Comment by sijie** [comment]
run cpp tests

**Pull request merged**